source/extensions/filters/http/common/jwks_fetcher.cc

```diff
@@ -106,19 +146,75 @@ class JwksFetcherImpl : public JwksFetcher,
   Upstream::ClusterManager& cm_;
   bool complete_{};
   JwksFetcher::JwksReceiver* receiver_{};
-  const envoy::config::core::v3::HttpUri* uri_{};
+
   Http::AsyncClient::Request* request_{};
 
+  Tracing::Span* parent_span_{};
+
+  const envoy::extensions::filters::http::jwt_authn::v3::RemoteJwks& remote_jwks_;
```

Comment:
we may not need to store remote_jwks, only need to store uri_.  Its retry_policy is used at constructor

Reply:
done